test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,10 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNullTest() {
+        assertFalse(Utils.isAnyNull("a", "b", "c"));
+        assertTrue(Utils.isAnyNull(null, "b", "c"));
```

Comment:
Good! You've included both positive and negative test cases.

Can do with more test cases on other scenarios such as `empty` collection, other data type etc as well.